ras-onboarding/ras_onboarding/namespace/queries.py

```
@@ -11,8 +23,61 @@
                                updated_by = 'system' \
                            """
 
+
+GET_NAMESPACES_FOR_DOMAIN = """
+            SELECT {id_column} as id,
+                   {namespace_column} as namespace,
+                   {domain_column} as domain_id
+            FROM {table}
+            WHERE {domain_column} = %s
+        """
+
+CREATE_NAMESPACE_BUNDLE = """
+            INSERT INTO bundle (id, name, description, owner_id, owner_type, domain,
+                               created_at, created_by, updated_at, updated_by, is_archived)
+            VALUES (%s, %s, %s, %s, %s, %s, current_timestamp, 'system', current_timestamp, 'system', false)
+        """
+
+GET_DOMAIN_MEMBERS = """
+            SELECT created_by
+            FROM domain
+            WHERE id = %s
+            LIMIT 1
+        """
+
+CHECK_NAMESPACE_IN_BUNDLE = """
+            SELECT 1 FROM bundle_asset
+            WHERE bundle_id = %s
+              AND asset_type = 'NAMESPACE'
+              AND asset_id = %s
+            LIMIT 1
+        """
+
+CHECK_NAMESPACE_IN_ANY_BUNDLE = """
+            SELECT b.id, b.name FROM bundle_asset ba
+            JOIN bundle b ON ba.bundle_id = b.id
+            WHERE ba.asset_type = 'NAMESPACE'
+              AND ba.asset_id = %s
+            LIMIT 1
+        """
+
+ADD_NAMESPACE_TO_BUNDLE = """
+            INSERT INTO bundle_asset (bundle_id, asset_type, asset_id, created_at, created_by, updated_at, updated_by)
+            VALUES (%s, 'NAMESPACE', %s, current_timestamp, 'system', current_timestamp, 'system')
+            ON CONFLICT (bundle_id, asset_type, asset_id) DO NOTHING
+        """
+
+DELETE_NAMESPACE_BUNDLE = """
+DELETE FROM bundle
+    WHERE id = %s"""
+
 DELETE_BUNDLE_PERMISSIONS = """
                             DELETE
                             FROM bundle_permission
                             WHERE bundle_id = %s \
```

i believe the tailing slash can be removed... did this query run properly? i believe it should throw a syntax error right? 🤔

---

In Python triple-quoted strings, a backslash at the end of a line does work as a line continuation - it removes the newline and joins the lines together.

But it is not required, so removing \ from the query.